firefox-ios/Client/Application/AppLaunchUtil.swift

```diff
@@ -40,6 +40,12 @@ final class AppLaunchUtil {
         // Need to get "settings.sendCrashReports" this way so that Sentry can be initialized before getting the Profile.
         let sendCrashReports = NSUserDefaultsPrefs(prefix: "profile").boolForKey(AppConstants.prefSendCrashReports) ?? true
 
+        if termsOfServiceManager.isAffectedUser {
+            logger.setup(sendCrashReports: sendCrashReports)
+            TelemetryWrapper.shared.setup(profile: profile)
+            TelemetryWrapper.shared.recordStartUpTelemetry()
+        }
+
         if termsOfServiceManager.isFeatureEnabled {
```

Comment:
Should this be an else? It's going to double call some of these functions. I think `recordStartUpTelemetry`() is the only one that might be problematic to fire multiple times but I'm not certain.

Reply:
For context, we already have an else that handles users without the feature enabled, and affected users might follow the feature enabled path. I’m not sure if you’re suggesting we add another else there, place it elsewhere, or do something different?